RELEASE.md

```diff
@@ -7,22 +7,33 @@ Please follow the established format:
 - Use present tense (e.g. 'Add new feature')
 - Include the ID number for the related PR (or PRs) in parentheses
 -->
+# Release 3.15.0
 
 ## Major features and improvements
 
-- Add `--autoreload` flag to `kedro viz` to automatically reload Kedro Viz tab when the Kedro project code changes.
+- Visualise related data nodes of a modular pipeline in focus mode (#530)
+
+## Bug fixes and other changes
+
+- Fix the display of transcoded data nodes and their metadata (#491, #525)
+- Remove `newparams` flag (#528)
+- Add notice about Kedro-UI deprecation on the Styleguide (#529)
+- Add more eslint rule (#532)
```

Comment:
Fixed the parameters-indicator hover tooltip to show parameter names when they are hidden. (?)

Reply:
Which PR is that?

Reply:
Found it